SPECS/procps-ng/procps-ng.spec

```diff
@@ -1,35 +1,40 @@
 Summary:        Programs for monitoring processes
 Name:           procps-ng
-Version:        3.3.15
-Release:        5%{?dist}
-License:        GPLv2
-URL:            http://procps.sourceforge.net/
-Group:          Applications/System
+Version:        3.3.17
+Release:        1%{?dist}
+License:        GPLv2 AND LGPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
+URL:            http://procps.sourceforge.net/
 Source0:        http://sourceforge.net/projects/procps-ng/files/Production/%{name}-%{version}.tar.xz
```

Comment:
can we use https://sourceforge.net/projects/procps-ng/files/Production/procps-ng-3.3.17.tar.xz  ?

Reply:
Fixed